BoxContentSDK/BoxContentSDK/QueueManagers/BOXParallelAPIQueueManager.m

```diff
@@ -51,6 +51,10 @@ - (id)initWithSession:(BOXAbstractSession *)session
         _uploadsQueue.name = @"BOXParallelAPIQueueManager upload queue";
         _uploadsQueue.maxConcurrentOperationCount = 2;
 
+        _smallDownloadsQueue = [[NSOperationQueue alloc] init];
+        _smallDownloadsQueue.name = @"BOXParallelAPIQueueManager small downloads queue";
+        _smallDownloadsQueue.maxConcurrentOperationCount = NSOperationQueueDefaultMaxConcurrentOperationCount;
```

Comment:
Yeah this is weird. We've been using this value in Bender, and I don't really notice a performance difference between our iOS 8.3 device and an iOS 9. My guess would be that this removes the limitation after iOS 8.4 and allows us to run as many request as we want in parallel.